egs/wsj/s5/steps/dur_model/get_examples.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+cmd=run.pl
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 2 ]; then
+   echo "Usage: $0 [options] <ali-dir> <duration-model-dir>"
+   echo "e.g.: $0 exp/mono_ali exp/mono/durmod"
+   exit 1;
+fi
+
+alidir=$1
+dir=$2
+
+durmodel=$dir/durmodel.mdl
+transmodel=$alidir/final.mdl
+
+for f in $transmodel $alidir/ali.1.gz; do
+  [ ! -f $f ] && echo "$0: Required file not found: $f" && exit 1;
+done
+
+if [ ! -f $dir/all.ali ]; then
+  gunzip -c $alidir/ali.*.gz >> $dir/all.ali || exit 1;
+fi
+
+if [ ! -f $dir/all.egs ]; then
+$cmd $dir/log/durmod_make_examples.log \
+     nnet3-durmodel-make-egs $durmodel $transmodel ark:$dir/all.ali ark,t:$dir/all.egs || exit 1;
+fi
+
+numparts=20
```

Comment:
In egs/wsj/s5/steps/dur_model/get_examples.sh
> <https://github.com/kaldi-asr/kaldi/pull/414#discussion_r49951639>:
>
> > +transmodel=$alidir/final.mdl
> > +
>
> Yes, it was unstable and I am not sure why. I need to complete my
> experiments.
>
> I don't think you should explicitly tune on the WER in the way you
> suggest-- it contaminates the test set too much.
>
> Could you please explain more? I did not understand what you mean by
> 'contaminate'
>

I mean that if you do too much detailed and systematic tuning on a test
set, it can no longer really be considered a test set, because it's as if
you've trained on it.

> I think it would be good if you could create at least one run on the CLSP
> cluster on some dataset we use-- this will enable me to look at your setup
> and help tune things.
>
> What dataset would you suggest and how should I do it?
>

How about Switchboard?  You could just run it with a GMM system, since it's
independent of the model type.   Make a directory in /export/[something],
instead of using your home directory.
Dan




> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/414/files#r49951639>.
>